docs/ble.md

```diff
@@ -58,157 +72,171 @@ dictionary CharacteristicInit {
     NotifyCallback onNotify;            // optional
 };
 
-callback ReadCallback = void (unsigned long offset, FulfillReadCallback);
+interface Characteristic {
+    attribute ReadCallback onReadRequest;
+    attribute WriteCallback onWriteRequest;
+    attribute SubscribeCallback onSubscribe;
```

Comment:
Very un-web like... should be `onsubscribe` all lowercase plus these should be nullable.

`attribute SubscribeCallback? onsubscribe`